src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/proteins-list-item-info/ProteinsListItemInfo.tsx

```diff
@@ -262,9 +274,108 @@ const ProteinExternalReference = (props: ProteinExternalReferenceProps) => {
 
   return (
     <div className={styles.proteinExternalReference}>
-      <ExternalReference label={props.source} to={url} linkText={props.name} />
+      <ExternalReference
+        label={props.source}
+        to={url}
+        linkText={props.accessionId}
+      />
     </div>
   );
 };
 
+type ProteinExternalReferenceGroupProps = {
+  source: ExternalSource;
+  xrefs: Array<
+    Pick<ExternalReferenceType, 'accession_id' | 'name'> &
+      Pick2<ExternalReferenceType, 'source', 'id'>
+  >;
+};
+
+const ProteinExternalReferenceGroup = (
+  props: ProteinExternalReferenceGroupProps
+) => {
+  const { source, xrefs } = props;
+
+  const [isXrefGroupOpen, setXrefGroupOpen] = useState(false);
+
+  const toggleXrefGroup = () => {
+    setXrefGroupOpen(!isXrefGroupOpen);
+  };
+
+  const chevronClass = classNames(
+    styles.chevron,
+    isXrefGroupOpen? styles.chevronDown : styles.chevronUp
+  );
+
+  if (xrefs.length > 3) {
+    const displayXref = xrefs[0];
+
+    if (!displayXref) {
+      return null;
+    }
+
+    return (
+      <div className={styles.xrefGroupWrapper}>
+          {!isXrefGroupOpen ? (
+            <div className={styles.xrefWithChevron}>
+              <ProteinExternalReference
+                key={displayXref.accession_id}
+                source={source}
+                accessionId={displayXref.accession_id}
+                name={displayXref.name}
+              />
+              <div className={styles.xrefGroupChevron} onClick={toggleXrefGroup}>
+                + {xrefs.length - 1}
+                <ChevronDown className={chevronClass} />
+              </div>
+            </div>
+          ) : (
+            <div className={styles.xrefGroupWrapper}>
+              <div className={styles.xrefWithChevron}>
+                <ProteinExternalReference
+                  key={displayXref.accession_id}
+                  source={source}
+                  accessionId={displayXref.accession_id}
+                  name={displayXref.name}
+                />
+                <div className={styles.xrefGroupChevron} onClick={toggleXrefGroup}>
+                  + {xrefs.length - 1}
+                  <ChevronDown className={chevronClass} />
+                </div>
+              </div>
+              { xrefs.map((xref, i) => {
+                if (i === 0) { // Ignore first element as it is displayed above with chevron.
+                  return;
+                }
```

Comment:
Instead of mapping over all xrefs and ignoring the first one, create a slice that excludes the first xref and map over it (`xrefs.slice(1).map(...)`)